src/ray/rpc/client_call.h

```diff
@@ -187,14 +188,16 @@ class ClientCallManager {
   void PollEventsFromCompletionQueue(int index) {
     void *got_tag;
     bool ok = false;
-    auto deadline = gpr_inf_future(GPR_CLOCK_REALTIME);
+    auto deadline = gpr_time_from_millis(100, GPR_CLOCK_REALTIME);
```

Comment:
Didn't you say the deadline time doesn't matter if we use the `shutdown_` flag? If so, can we revert this to avoid unnecessary wake ups?

Reply:
I later saw it hang without this timeout.